pyof/foundation/basic_types.py

```diff
@@ -430,8 +435,10 @@ def pack(self, value=None):
         if value is None:
```

Comment:
This line + line 438 could'n appear as 
`if not value:`
and then
`else`

or the other way around?